### PR TITLE
Fixes to run experiments

### PR DESCRIPTION
**Pull request by beatrizleon**
* Moving here scenes and queries
* Creating experiments
* Having an easy way to launch the robot

### CODE REVIEW

File: sr_moveit_planner_benchmarking/CMakeLists.txt [hunk 1]
```diff
@@ -27,8 +27,8 @@ install(DIRECTORY launch DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
 
 ## Add folders to be run by python nosetests
 # catkin_add_nosetests(test)
-if (CATKIN_ENABLE_TESTING)
-    find_package(rostest REQUIRED)
-    add_rostest(test/test_import_export_scenes.test)
-    add_rostest(test/test_import_export_queries.test)
-endif ()
\ No newline at end of file
+# if (CATKIN_ENABLE_TESTING)
```
Comment (mikramarc) [inline sr_moveit_planner_benchmarking/CMakeLists.txt:30]: why have there tests been commented out?

File: sr_moveit_planner_benchmarking/experiments/launch/benchmarking_parameters.launch [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<launch>
```
Comment (mikramarc) [inline sr_moveit_planner_benchmarking/experiments/launch/benchmarking_parameters.launch]: this launch file is a duplicate of benchmark_scene_ground_with_boxes.launch

File: sr_moveit_planner_benchmarking/experiments/launch/load_all_scenes_and_queries_to_db.launch [hunk 3]
```diff
@@ -0,0 +1,54 @@
+<launch>
+    <!-- Load robot and warehouse-->
```
Comment (mikramarc) [inline sr_moveit_planner_benchmarking/experiments/launch/load_all_scenes_and_queries_to_db.launch:2]: should we cange all the "default" to "value" if we dont parametrize from outside? seems a bit insonsistent when some of them are "value" and some "defualt"

**Review by mikramarc — CHANGES_REQUESTED** [review]

File: sr_moveit_planner_benchmarking/CMakeLists.txt (hunk 1 above)
Comment (beatrizleon) [reply sr_moveit_planner_benchmarking/CMakeLists.txt:30]: They have been commented until mongo db images work on jenkins again

File: sr_moveit_planner_benchmarking/experiments/launch/benchmarking_parameters.launch (hunk 2 above)
Comment (beatrizleon) [reply sr_moveit_planner_benchmarking/experiments/launch/benchmarking_parameters.launch]: We can remove this

File: sr_moveit_planner_benchmarking/experiments/launch/load_all_scenes_and_queries_to_db.launch (hunk 3 above)
Comment (beatrizleon) [reply sr_moveit_planner_benchmarking/experiments/launch/load_all_scenes_and_queries_to_db.launch:2]: Yes please change

Comment (beatrizleon) [reply sr_moveit_planner_benchmarking/experiments/launch/load_all_scenes_and_queries_to_db.launch:2]: Yes please change

**Review by mikramarc — APPROVED** [review]

**Pull request merged**